pkt-line.c

```diff
@@ -9,7 +9,14 @@
  */
 char packet_buffer[LARGE_PACKET_MAX];
 
+/*
+ * Warning: `packet_trace_prefix` and the static variables `in_pack`
+ * and `sideband` inside of `packet_trace()` make packet tracing not
+ * safe for threaded or concurrent operations.
+ * TODO Eventually, we should revisit this.
+ */
 static const char *packet_trace_prefix = "git";
+
```

Comment:
It's worth pointing out that `packet_trace_prefix` is actually modified by `packet_trace_identify(const char *)` [just outside the diff context](https://github.com/microsoft/git/pull/274/commits/0bf630772df590fe8bab62dfbad239596f2adcf5#diff-01dd8ef8237fa17ca57615106451a81fR25).

Reply:
Right. The trace stuff is something specifically left out from this patch series. @jeffhostetler looked into it, and I repeated the exercise, and yes, the changes required to make this thread-safe are horrendous.